test/performance/Readme.md

```diff
@@ -0,0 +1,21 @@
+# Performance Tests using k6
+
+This Note will walk you through the process of setting up the k6 
```

Comment:
Add 2-3 sentences about k6.

Add a Prerequisites section which mentions k6 and a link to their docs page.

in `Available tests` write a short summary what a test does